README.md

```diff
@@ -28,7 +28,7 @@ Available addons
 addon | version | maintainers | summary
 --- | --- | --- | ---
 [openupgrade_framework](openupgrade_framework/) | 16.0.1.0.0 | [![legalsylvain](https://github.com/legalsylvain.png?size=30px)](https://github.com/legalsylvain) [![StefanRijnhart](https://github.com/StefanRijnhart.png?size=30px)](https://github.com/StefanRijnhart) [![hbrunn](https://github.com/hbrunn.png?size=30px)](https://github.com/hbrunn) | Module to integrate in the server_wide_modules option to make upgrades between two major revisions.
-[openupgrade_scripts](openupgrade_scripts/) | 16.0.1.0.3 |  | Module that contains all the migrations analysis and scripts for migrate Odoo SA modules.
+[openupgrade_scripts](openupgrade_scripts/) | 16.0.1.0.3 |  | Module that contains all the migrations analysis and scripts for migrating Odoo SA modules.
```

Comment:
This must be corrected here: https://github.com/OCA/OpenUpgrade/blob/16.0/openupgrade_scripts/__manifest__.py#L6, as this part is auto-generated.

Reply:
Done. Thank you !